### PR TITLE
Added the popup window functionality

### PR DESCRIPTION
**Pull request by AshwinCarvalho999**
Added index.html
Added style.css
Added Linters
Added script.js
Added  style.css
Added Popup Window for card section so that users can interact and navigate all details of the project
Added .gitignore file to ignore node_modules file
Added Readme file

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+    highlight:
+      "1 - A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    highlightDetails:
+      "Highlight Featured is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five, but also the leap into electronic typesetting, remaining essent",
+    image: 'images/project-featured.jpg',
+    badge1: 'css',
+    badge2: 'html',
+    badge3: 'Bootstrap',
+    badge4: 'Ruby',
+  },
```
Comment (FrederickMih) [inline script.js:16]: - [ ] Based on the requirements of this project, the popup should follow the design in Figma for both desktop and mobile versions. Please compare the screenshots below for the desktop versions and make the necessary changes. You'll need to align those buttons and text exactly as seen in the original screenshot below. 

| Yours | Figma |
| --- | --- |
| Screenshot from 2022-09-06 16-06-24 | Screenshot from 2022-09-06 16-02-23 |

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @AshwinCarvalho999 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- No Linter error :+1: 
- PR has a title and summary :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -0,0 +1,1687 @@
...
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  background-color: white;
+  color: #e05304;
+  width: 59%;
+  position: relative;
+  margin: 2% 0;
+  gap: 16px;
+}
```
Comment (codepantha) [inline style.css:449]: - [ ] While implementing the required changes for the popup modal on desktop devices, you have broken the popup modal design for mobile devices. Kindly ensure that the width on mobile devices corresponds to the given figma design template 👍 
## Your Previous Design

## Your Current Design
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi @AshwinCarvalho999 👋, 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.

## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ Good UI

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AshwinCarvalho999** [comment]
Done.

**Comment by AshwinCarvalho999** [comment]
Awesome, thanks!

**Pull request merged**